server/recceiver/cfstore.py

```diff
@@ -175,9 +211,34 @@ def checkPropertiesExist(client, propOwner):
     '''
     requiredProperties = ['hostName', 'iocName', 'pvStatus', 'time']
     for propName in requiredProperties:
-        if client.findProperty(propName) == None:
+        if client.findProperty(propName) is None:
             try:
-                client.set(property={u'name': propName, u'owner':propOwner})
+                client.set(property={u'name': propName, u'owner': propOwner})
             except Exception as e:
-                _log.error('Failed to create the property %s: %s',propName, e)
+                _log.error('Failed to create the property %s: %s', propName, e)
+
+
+def getCurrentTime():
+    return str(datetime.datetime.now())
+
+
+def poll(update, client, new, delrec, channels_dict, iocs, hostName, iocName, times, owner):
+    sleep = 1
+    success = False
+    while not success:
+        try:
+            update(client, new, delrec, channels_dict, iocs, hostName, iocName, times, owner)
+            success = True
+            return success
+        #except requests.exceptions.HTTPError as e:  # should catch only network errors
+        except StandardError as e:
```

Comment:
What is the significance of ```StandardError``` here?

FYI, A tuple of types may be given here.  ie:

```py
    except (ValueError, TypeError) as e:
```